packages/@aws-cdk/aws-redshift/lib/private/database-query-provider/types.ts

```diff
@@ -0,0 +1,50 @@
+import { DatabaseQueryHandlerProps, TableHandlerProps } from '../handler-props';
```

Comment:
The ENUMs are needed to be defined here since they are used in the `database-query=provider/table.ts` and if they are not defined in the `database-query-provider` folder then they are not included in the custom-resource.